azure-pipelines.yml

```diff
@@ -19,10 +19,13 @@ steps:
     echo "##vso[task.setvariable variable=LD_LIBRARY_PATH]$lib_path"
   displayName: Update LD_LIBRARY_PATH for GitVersion
 
-- task: GitVersion@4
-  displayName: GitVersion
+- task: gitversion/setup@0
+  displayName: Install GitVersion
   inputs:
-    updateAssemblyInfo: false
+    versionSpec: '5.6.0'
```

Comment:
Good to see that there is a fix, one question though - can versionspec be set to latest just like the vmImage?

Reply:
going through the doc, I think it was recommended to use this version for now, they are planning for a new task which will be released soon and currently is in preview state, so we can think of updating it then

Reply:
however we are moving to SDS not sure we will even be using this tasks anymore when we do that

Reply:
OK.  Thanks.